ucentral.state.pretty.json

```diff
@@ -158,6 +158,103 @@
                     }
                 }
             }
+        },
+        "stp": {
+            "description": "Global status and configuration of the Spanning Tree Protocol (STP) across the switch. Supports variants like STP, RSTP, MSTP. Includes bridge-wide parameters and protocol behavior.",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "type": "boolean",
+                    "description": "Whether STP is enabled globally on the switch."
+                },
+                "mode": {
+                    "type": "string",
+                    "description": "Operational STP mode used across the switch.",
+                    "enum": [
+                        "STP",
+                        "RSTP",
+                        "MSTP",
+                        "PVST",
+                        "RPVSTP"
+                    ]
+                },
+                "transmission-limit": {
+                    "type": "integer",
+                    "description": "Maximum number of BPDUs that can be transmitted per hello time."
+                },
+                "path-cost-method": {
+                    "type": "string",
+                    "description": "Indicates whether \"Short\" or \"Long\" path cost method is in use.",
+                    "enum": [
+                        "Short",
+                        "Long"
+                    ]
+                },
+                "max-hops": {
+                    "type": "integer",
+                    "description": "Maximum hop count used in MSTP."
+                },
+                "instances": null,
```

Comment:
See comment re indentation in the state.yml file, this needs to be fixed in the yml and regenerated